src/cli-sdk/test/config/definition.ts

```diff
@@ -1,4 +1,6 @@
 import t from 'tap'
+import { getSortedCliOptions } from '../../src/config/definition.ts'
```

Comment:
This is why coverage is being detected incorrectly. All imports for a file need to be done with `t.mockImport` if there is top-level state being altered in the tests: https://github.com/vltpkg/vltpkg/blob/main/CONTRIBUTING.md#test-coverage-is-failing-but-it-shouldnt-be